mmv1/products/spanner/BackupSchedule.yaml

```diff
@@ -154,3 +154,27 @@ properties:
       - 'incrementalBackupSpec'
     properties:
       []
+  - name: 'encryptionConfig'
+    type: NestedObject
+    description: |
+      Configuration for the encryption of the backup schedule.
+    default_from_api: true
```

Comment:
Does this whole thing need default_from_api? What gets set if the user doesn't specify it?